core/src/main/scala/org/apache/spark/scheduler/OutputCommitCoordinator.scala

```diff
@@ -48,25 +48,28 @@ private[spark] class OutputCommitCoordinator(conf: SparkConf, isDriver: Boolean)
   private type StageId = Int
   private type PartitionId = Int
   private type TaskAttemptNumber = Int
+  private case class StageState(authorizedCommitters: Array[TaskAttemptNumber],
```

Comment:
can you put a comment here that the index into the `authorizedCommitters` array is the partitionId ?